src/main/webapp/components/result-forms/route.js

```diff
@@ -171,18 +198,35 @@ const resultForms = gql`
   query ResultForms {
     metacardsByTag(tag: "attribute-group") {
       attributes {
+        security_access_individuals_read
+        security_access_individuals
+        security_access_administrators
+        security_access_groups_read
+        security_access_groups
```

Comment:
Any reason not include these on the respective fragments for workspaces, search forms, and result forms?

Reply:
No, no reason I'll update it to use the fragment